### PR TITLE
WIP: Develop 1339 - update app to run on Python 3.6

### PR DESCRIPTION
**Pull request by mariya**
**What problems does this PR solve?**
The initial purpose of this PR was to update some vulnerable packages which were setting off Dependabot alerts.

In the process, I updated the app to run on Python 3.6 (3.5 is no longer supported), which required some changes to the tests, including mocking of the Redis client.

**How has the changes been tested?**
Only using the automated tests. This should be tested thoroughly on staging, where we have an actual PDC client.

**Reasons for careful code review**
If any of the boxes below are checked, extra careful code review should be inititated.

  - [ ] This PR contains code that could remove data

### CODE REVIEW

File: archive_verify/__init__.py [hunk 1]
```diff
@@ -1 +1 @@
-__version__ = "1.0.0"
+__version__ = "1.0.1"
```
Comment (matrulda) [inline archive_verify/__init__.py]: It looks like this version tag is a bit outdated: https://github.com/Molmed/snpseq-archive-verify/releases

**Review by matrulda — CHANGES_REQUESTED** [review]
Looks good! I only think the version tag needs to be updated.

File: archive_verify/__init__.py (hunk 1 above)
Comment (mariya) [reply archive_verify/__init__.py]: Good catch! Fixed now.

**Review by matrulda — APPROVED** [review]
Great!

**Pull request merged**